src/universal/components/Dashboard/dashTimestamp.js

```diff
@@ -0,0 +1,33 @@
+const days = [
```

Comment:
this probably belongs in a `utils` folder, since I imagine we'll use this for more components.